nat/service_ics.go

```diff
@@ -206,6 +219,57 @@ func (nat *serviceICS) getInternalInterfaceName() (string, error) {
 	return ifaceName, nil
 }
 
+func (nat *serviceICS) interfaceIPWatchdog() {
+	for {
+		select {
+		case <-time.After(5 * time.Second):
```

Comment:
Yeach, this looks really a hack :( We should search for help probably here. Maybe we can write to some windows forum or stackoverflow with this issue. We will not be able to be confident with this.

Reply:
@vkuznecovas suggest maybe to make this a bit more deterministic, we can check interface's IP right after it is configured? Did you test time span when this change from windows occurs?

Reply:
That problem is that IP is not assigned by ICS immediately after enabling sharing. It happens after some time. 
During my tests with manual IP assignment, it was not changed back after several hours. 
So, the current implementation sets it once and checking it every 5 seconds.

Reply:
The another possible way is to change ICS configuration in Windows registry to assigne our addresses through ICS. 
But we are limited to `/24` subnet using this way.

Reply:
Changed to the implementation for setting IP as ICS config.